README.md

```diff
@@ -1,299 +1,75 @@
-# CollabSwarm
+# SwarmDB
 
-[![Gitter](https://badges.gitter.im/collabswarm-dev/community.svg)](https://gitter.im/collabswarm-dev/community?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge)
+(name change in process; for future reference please use: https://swarmdb.dev/ which we will keep up to date even if the repo name changes)
 
-![Collaborative Editing Example](docs/collaborative-editing.gif)
```

Comment:
Would be nice to add back a gif of a working example at some point

Reply:
Or an actual working example?